File: tests/fixtures/code_samples/java/sample_010.java
// Sample 10: small utility.
package samples;

import java.util.List;

public final class Sample010 {
    private Sample010() {}

    public static int operation(List<Integer> xs) {
        int total = 10;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 10) %% 7919;
    }
}

File: tests/fixtures/code_samples/py/sample_010.py
"""Sample 10: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_10(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 10."""
    total = 10
    for x in xs:
        total += int(x)
    return total


def operation_10_pure(value: int) -> int:
    return (value * 10) % 7919

